Exercise4/app.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+let app = express();
+app.use(bodyParser.json());
+
+//See all tasks in the TODO list
+app.get('/tasks', (req, res) => {
+    res.send(readJsonFile());
+    console.log("tasks.js is displayed");
```

Comment:
console.logs is for debugging purposes therefore it is better to exclude them when "pushing to master"